visualization_config:
  magnetic_field_marker:
    field_scale: 0.005
    lifetime: 0.1
    arrow:
      shaft_diameter: 0.0005
      head_diameter: 0.0025
    color: [1.0, 1.0, 0.0, 1.0] # RGBA
  magnet_marker:
    scale:
      x: 0.001
      y: 0.001
      z: 0.002
    color: [1.0, 0.0, 0.0, 1.0]

color_map:
  max: 3.2
